CHANGES

```diff
@@ -6,6 +6,15 @@ Tactics
 - New tactic "extensionality in H" which applies (possibly dependent)
   functional extensionality in H supposed to be a quantified equality
   until giving a bare equality.
+- New tactic "inversion_sigma" which turns equalities of dependent
+  pairs (e.g., "existT P x p = existT P y q", frequently left over by
+  "inversion" on a dependent type family) into pairs of equalities
+  (e.g., a hypothesis "H : x = y" and a hypothesis of type "rew H in p
+  = q"); these hypotheses can subsequently be simplified using
+  "subst", without ever invoking UIP / axiom K.  For more fine-grained
+  control, you you can invoke "induction H as [H1 H2] using
```

Comment:
I can't see at once what "more fine-grained control" refers to, so maybe being more explicit about it.

Otherwise very good.